torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7917,6 +7936,42 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                # UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out_warning'),
            )),
+    OpInfo('uniform',
+           op=lambda inp, *args, **kwargs: wrapper_set_seed(torch.Tensor.uniform_, inp, *args, **kwargs),
+           method_variant=None,
+           inplace_variant=torch.Tensor.resize_,
+           dtypes=floating_and_complex_types_and(torch.bfloat16, torch.float16),
+           supports_out=False,
+           supports_autograd=False,
+           is_factory_function=False,
+           sample_inputs_func=sample_inputs_uniform,
+           error_inputs_func=error_inputs_uniform,
+           skips=(
+               # https://github.com/pytorch/pytorch/issues/81774
+               DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),
```

Comment:
The issue referenced above deals with a different test also it mentions function which take `TensorOptions`. Uniform does not appear to have options in the signature.

Reply:
Updated the comment, I'm getting this error `FX failed to normalize op - add the op to the op_skip list.` looking into it further, but I don't know what normalize the op is trying to do yet.

Reply:
This is the same error as the op `sum_to_size` is skipping

Reply:
These two tests:
```
            # lambda impl
            DecorateInfo(unittest.expectedFailure, 'TestNormalizeOperators', 'test_normalize_operator_exhaustive'),
            DecorateInfo(unittest.expectedFailure, 'TestJit', 'test_variant_consistency_jit'),),
```
fail on operations that are defined via a lambda, like yours (you are using a lambda to set the seed).
You can grep for these failures in the `common_methods_invocations.py` invocation to see this.

Reply:
Note that this is reported in the error:
```
RuntimeError: 
                    FX failed to normalize op - add the op to the op_skip list.
                    A common reason is if your OpInfo was implemented with a lambda
                    otherwise, file an issue
```

Reply:
👍